README.md

```diff
@@ -33,7 +33,7 @@ let h_new = bcrypt::hash("password").unwrap();
 // Verify a password against an existing hash.
 let h = "$2y$05$bvIG6Nmid91Mu9RcmmWZfO\
          5HJIMCT8riNW0hEp8f6/FuA2/mHZFpe";
-assert_eq!(bcrypt::verify("password", h), true);
+assert!(bcrypt::verify("password", h));
```

Comment:
Perhaps the example code with assert! would look nicer if "verify" was called "equals_password" or something like that (such that the code reads a bit more like english)

On 17 February 2016 18:52:11 CET, inejge <notifications@github.com> wrote:
>> @@ -33,7 +33,7 @@ let h_new = bcrypt::hash("password").unwrap();
>>  // Verify a password against an existing hash.
>>  let h = "$2y$05$bvIG6Nmid91Mu9RcmmWZfO\
>>           5HJIMCT8riNW0hEp8f6/FuA2/mHZFpe";
>> -assert_eq!(bcrypt::verify("password", h), true);
>> +assert!(bcrypt::verify("password", h));
>
>You're right, of course; `assert_eq!` is technically redundant here,
>but I deliberately chose that form to emphasize that I'm checking the
>return value of `verify()`. Nevertheless, I'll think some more about
>using bare `assert!`, and then decide whether to merge this change.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/inejge/pwhash/pull/1/files#r53202773

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.